docs/01-WhatIsLORISMRI.md

```diff
@@ -1 +1,12 @@
 # 1.0 - What is LORIS-MRI?
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. These documents assume you have some knowledge regarding LORIS and a functioning installation. For information regarding LORIS itself, please consult the [LORIS wiki][1].
```

Comment:
Can you fix the length of these lines? It's hard to read with everything on one line.

(the easiest way I know of to consistently fix it for paragraphs is to pipe the file through the unix `fmt` command..